kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/KuraInterfaceStatusTest.java

```diff
@@ -0,0 +1,75 @@
+package org.eclipse.kura.nm;
```

Comment:
Add the copyright header.

Reply:
Addressed in 4a93f050db92c5bd9193913c61ecbf03a775a2d4